pom.xml

```diff
@@ -159,6 +231,7 @@
             <groupId>org.hibernate</groupId>
```

Comment:
is hibernate really a compile time dependency? JPA should be our compile dependecy.

please take a deep look at all the dependencies as it is difficult to maintain them after the fact we introduced with wrong scope.

Reply:
@arvindn05, This was already there and I have not introduced this dependency. I also had a closer look at this and see that it is indeed compile time dependency as it is used in NetworkElementEntity, as it uses the following: org.hibernate.annotations.Fetch;org.hibernate.annotations.FetchMode; and also used in InspectionPortEntity: org.hibernate.annotations.GenericGenerator;

Reply:
@arvindn05, I had the closer look at the dependency I had put and see the following changes: javax.servlet-api, scope shall be changed to provided, jersey-server: scope shall be changed to provided,jersey-container-servlet-core: scope shall be changed provided,javax.ws.rs-api ; scope shall be changed to provided, jackson-annotations to 'runtime',jackson-core to runtime,jackson-databind to runtime,jackson-jaxrs-base to runtime, org.apache.commons.lang to provided.